restli-tools/src/test/resources/idls/twitter-statusPromises.restspec.json

```diff
@@ -20,42 +20,50 @@
     "supports" : [ "batch_get", "create", "delete", "get", "get_all", "partial_update" ],
     "methods" : [ {
       "method" : "create",
+      "javaMethodName" : "create",
       "javaMethodType" : "PROMISE",
       "doc" : "Creates a new Status"
     }, {
       "method" : "get",
+      "javaMethodName" : "get",
       "javaMethodType" : "PROMISE",
       "javaReturnsResultWrapper" : false,
       "doc" : "Gets a single status resource"
     }, {
       "method" : "partial_update",
+      "javaMethodName" : "update",
       "javaMethodType" : "PROMISE",
       "doc" : "Updates a single status resource"
     }, {
       "method" : "delete",
+      "javaMethodName" : "delete",
       "javaMethodType" : "PROMISE",
       "doc" : "Deletes a status resource"
     }, {
       "method" : "batch_get",
+      "javaMethodName" : "batchGet",
       "javaMethodType" : "PROMISE",
       "javaReturnsResultWrapper" : false,
       "doc" : "Gets a batch of statuses"
     }, {
       "method" : "get_all",
+      "javaMethodName" : "getAll",
       "javaMethodType" : "PROMISE",
       "javaReturnsResultWrapper" : false,
       "doc" : "Gets all the resources",
       "pagingSupported" : true
     } ],
     "finders" : [ {
       "name" : "user_timeline",
+      "javaMethodName" : "getUserTimeline",
       "javaMethodType" : "PROMISE",
       "javaReturnsResultWrapper" : false,
       "doc" : "Gets the status timeline for a given user",
       "pagingSupported" : true
     } ],
     "batchFinders" : [ {
       "name" : "batchFinderByAction",
+      "javaMethodName" : "batchFindStatuses",
```

Comment:
This name differs from pojo (StatusCollectionResource.java) method name as well as BatchFinder value, is this expected?
```
  @BatchFinder(value="batchFinderByAction",  batchParam="action")
  public BatchFinderResult<Status, Status, EmptyRecord> batchFinderByAction(
      @QueryParam("action") Status[]  actionNames,
      @QueryParam("q") @Optional String qParam
  )
  {
    return null;
  }
```

Reply:
This IDL is generated from `StatusPromiseResource`, which has a method like this:

```
  @BatchFinder(value="batchFinderByAction",  batchParam="criteria")
  public Promise<BatchFinderResult<Status, Status, User>> batchFindStatuses(@QueryParam("criteria") Status[] criteria)
  {
    return null;
  }
  ```

Reply:
thanks. Didn't refresh my repo so I was looking at the wrong file.